docs/webhooks/Getting-Started.md

```diff
@@ -0,0 +1,55 @@
+---
+tags: [Webhooks]
+---
+
+# Getting Started
+
+## Tunnelling traffic to your local machine
+
+To test out webhooks locally without deploying an application to an environment or exposing your test environments to the Internet, you can use ngrok, a free service which tunnels internet traffic to your local environment. 
+
+Download ngrok [here](https://ngrok.com/download), then follow their getting started guide.
+
+## Configuring webhooks through the portal
+
+Once you’re up and running with ngrok, you’ll need to configure our webhooks as follows:
+
+1. Generate a secret in the portal and save its value for later
+
+  ![Generate a secret in the portal](../../assets/images/webhooks/Generate-Secret.png)
+
+2. Create subscription in the portal, using the HTTPS URL provided by ngrok, e.g. `https://9999-123-456-789-12.ngrok.io`
```

Comment:
We should be explicit that a path would likely be used e.g. `https://9999-123-456-789-12.ngrok.io/webhooks`